src/data/hero.ts

```diff
@@ -8,11 +20,38 @@ export const getHeroAbilities = (
   case "Captain Zane":
     switch (index) {
     case 0:
-      return [ "TODO" ];
+      return [ HASTE ];
     case 1:
-      return [ "TODO" ];
+      return [
+        event("KILLS", ($, I, e) => {
+          const P = getPlayer($, I.controller);
+          if (P == null) {
+            return;
+          }
+
+          if (isScavenger($, e.instance)) {
```

Comment:
He only gets a coin or a card if he is the one who kills the patroller, right? Also, does there need to be any check to ensure that the owner of Zane is not the owner of the patroller?

Reply:
Since this event doesn't fire yet this is a null point, but it ends up depending on how these events are defined. Either the `KILLS` event should only fire for the instance that actually makes the kill, or it should include a pointer / ID for the killer. Which event would be more useful would maybe need some analysis of cards to determine (though ofc this will need to be decided later).

In general, my current leaning is that I will probably make all the events as explicit as possible: for example, `THIS_ARRIVES` and `OTHER_ARRIVES` will most definitely be separate events.

Reply:
Also also, w.r.t. this question:

> does there need to be any check to ensure that the owner of Zane is not the owner of the patroller?

- Nitpicky but worth pointing out, controller should matter not owner (it's important to be careful with this distinction because any bugs we introduce between the two would be really subtle and rare)
- By definition the active player can't have any patrollers (patroller support doesn't exist yet ofc, which means there are also no safeguards for this, but there should be in the future or it will cause lots of other problems too)